tests/unit/test_notebooks_pyspark.py

```diff
@@ -9,6 +9,9 @@
 
 @pytest.mark.notebooks
 @pytest.mark.spark
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="Takes 1087.56s in Windows, while in Linux 52.51s"
```

Comment:
Did we find a clue why on Windows it takes such long time?

Reply:
I think it is related to a configuration issue, I tried to debug it but didn't spend too much time. I left the notice there in case someone in the future wants to follow up with it